sklearn/cluster/_bicluster.py

```diff
@@ -139,22 +139,21 @@ def _svd(self, array, n_components, n_discard):
                                       **kwargs)
 
         elif self.svd_method == 'arpack':
-            u, _, vt = svds(array, k=n_components, ncv=self.n_svd_vecs)
+            v0 = _init_arpack_v0(min(array.shape), self.random_state)
```

Comment:
We can check for the random state here and pass along to all future calls to `_init_arpack_v0` in this method.

Reply:
Done. Written check_random_state as first statement in 'arrack' elif block and used it in all future calls to _init_arpack_v0